### PR TITLE
[Tabs extended] add modifier section

### PR DESCRIPTION
**Pull request by shixiedesign**
### Related Ticket(s)

#1217

### Changelog

**New**

- Added Modifier section, and the `Orientation` modifier. Added image for when Tab orientation is vertical. 

**Removed**

- duplicated image files of `Tabs extended with media`

image

### CODE REVIEW

File: src/pages/components/tabs-extended.mdx [hunk 1]
```diff
@@ -43,13 +43,29 @@ On mobile, Tabs extended switches to an accordion for easier browsing.
...
 <Caption>Example of the tabs extended, on mobile</Caption>
 
 </Column>
 </Row>
 
+### Modifiers
+
+#### Orientation
+
+Tabs orientation on desktop can be set to horizontal or vertical. When using vertical, the tabs are displayed stacked in 1 column. This orientation allows for longer tab labels.
```
Comment (RichKummer) [inline src/pages/components/tabs-extended.mdx:57]: I would rework this a little bit:

"Tabs orientation on desktop can be set to horizontal or vertical. When using the vertical orientation, the tabs are displayed stacked in one column. This allows for longer tab labels."

**Review by RichKummer — APPROVED** [review]

**Comment by DragosRistici** [comment]
@shixiedesign  Hey, Shix, its great anyway, but just to poke the bear, here's my little stuff: 

1. Maybe this image should be 2:1 and if you heighten the gray container, you could get a nice top-left-bottom padding of 48 maybe. This image ratio seems too narrow?
Screenshot 2021-12-03 at 16 26 30

2. From time to time maybe refer to the component, as in "the Tabs extended component", im just saying because tabs extended is plural and in fact there's a singular component.

3. Here the same as 1, the ratio seems over the 2:1
Screenshot 2021-12-03 at 16 30 07

4. I would add a little guidance of when to use it and when not to use it, too, as I know some cases where tabs and switchers were mixed and confused when used. 

Just saying, these are little things, not that important, maybe in the future, now overall this looks amazingly good!!!

File: src/pages/components/tabs-extended.mdx (hunk 1 above)
Comment (janchild) [reply src/pages/components/tabs-extended.mdx:57]: I like your rework, @RichKummer

Comment (janchild) [inline src/pages/components/tabs-extended.mdx:57]:
```suggestion
Tabs orientation on desktop can be set to horizontal or vertical. When using the vertical orientation, the tabs are displayed stacked in one column. This allows for longer tab labels.
```

**Review by janchild — COMMENTED** [review]
@RichKummer over to you!

**Comment by shixiedesign** [comment]
Thanks for all the feedback! I'm bringing back Dragos's magenta tick marks to denote grid. They measure 1px by 8px, and every 4 columns.

**Pull request merged**